engine/gui/src/gui.cpp

```diff
@@ -3874,6 +3875,18 @@ Result DeleteDynamicTexture(HScene scene, const dmhash_t texture_hash)
         }
     }
 
+    bool IsNodeVisible(HScene scene, HNode node)
```

Comment:
Our naming convention is "SetNode.../GetNode..."

Reply:
ah, ok